admin/scripts/modules/aktivity/prihlaseni-drd.php

```diff
@@ -20,15 +20,29 @@
     $mail->predmet('Gamecon: umístění družiny v MDrD');
     $druzina = $a->tym()->nazev();
 
+    // E-maily účastníkům
     foreach ($a->prihlaseni() as $uc) {
-        $mail->text("Ahoj " . $uc->nick() . ",\n
+        $mail->text(
+            "Ahoj " . $uc->nick() . ",\n
             bohužel, tvoje družina " . $druzina . " nepostoupila do dalšího kola. Herní bloky, původně 
             rezervované pro turnaj MDrD, jsou nyní volné a můžeš se tak přihlásit na jinou aktivitu.\n
             Díky a s pozdravem,\n 
             tým MDrD");
         $mail->adresat($uc->mail());
         $mail->odeslat();
     }
+
+    // E-mail PJovi
+    foreach (Uzivatel::zIds($a->getOrganizatoriIds()) as $pj) {
+        $mail->text(
+            "Ahoj, " . ",\n
```

Comment:
Jestli jsem něco neminul, tak Guff chtěl znění trochu jinak:

```
"Ahoj [Nick], tvoje čtvrtfinálová družina [Jméno družiny] nepostoupila do další fáze turnaje MDrD (vypadla v [Název kola, které odehráli]).

Toto je informační mail, který obdrželi i hráči z družiny.

S pozdravem, tým MDrD."
```

viz https://trello.com/c/q1Gdu1lG/888-mdrd-hromadn%C3%A9-informov%C3%A1n%C3%AD-dru%C5%BEin-o-nepostupu#comment-61b9de17bec8a2256d02cfe2

Takže by to mohlo být nějak jako

```php
$mail->text(<<<TEXT
Ahoj {$pj->jmenoNick()}, tvoje čtvrtfinálová družina {$druzina} nepostoupila do další fáze turnaje MDrD (vypadla v {$aVypadli->nazev()}).

Toto je informační mail, který obdrželi i hráči z družiny.

S pozdravem, tým MDrD.
TEXT
);
```
To podivné <<<TEXT je heredoc https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc který je užitečný v tom, že drží nové řádky a nemusí se tam psát `\n`. Zároveň se v tom dají vypisovat hodnoty z funkcí, podobně jako ve dvojitých uvozovkách, přes složené závorky `{$trida->funkce()}`.

Reply:
Díky, to jsem neznal a rád jsem se přiučil. :+1: 
Ten text mailu jsi pochopil dobře, díky za postřeh, já to úplně přehlídl, tak jsem improvizoval.